Sources/Testing/Testing.docc/Documentation.md

```diff
@@ -37,21 +37,38 @@ concurrency, and parameterize test functions across wide ranges of inputs.
 
 ## Topics
 
-### Getting started
+### Essentials
 
 - <doc:TemporaryGettingStarted>
-- <doc:MigratingFromXCTest>
-
-### Defining tests
-
 - <doc:DefiningTests>
 - <doc:OrganizingTests>
+- ``Test(_:_:)``
+- ``Test``
+- ``Suite(_:_:)``
+
+### Parameterized tests
+
 - <doc:ParameterizedTesting>
+- ``Test(_:_:arguments:)-8kn7a`` <!-- @attached(peer) macro Test<C>(_ displayName: String? = nil, _ traits: any TestTrait..., arguments collection: C) where C : Collection, C : Sendable, C.Element : Sendable -->
```

Comment:
This is not a blocker for this PR, but I wonder if we should just make a list somewhere of what the various mangled symbol names mean…